src/app/[locale]/page.tsx

```diff
@@ -12,19 +19,41 @@ const COMPONENTS = [
   { id: 1, component: <Categories /> },
   { id: 2, component: <Advantages /> },
   { id: 3, component: <AboutApp /> },
-  { id: 4, component: <ExpertsFeed /> },
+  {
+    id: 4,
+    component: (top_experts: UserSearchOut[]) => (
+      <ExpertsFeed>
+        <ExpertsCarousel experts={top_experts} />
+      </ExpertsFeed>
+    ),
+  },
   { id: 5, component: <HowToStart /> },
   { id: 6, component: <Faq /> },
 ];
 
-export default function Home() {
+export default async function Home() {
+  const { aPIGetPublicJobStatistics } = getJobs();
+  const { aPIGetPublicTopExperts } = getUsers();
+  const {
```

Comment:
Why can't `UkraineMap` do data fetching inside itself?

Reply:
Same answer as above. All these components are on the same page, so I don't see the point in doing full encapsulation.